ophyd/epics_motor.py

```diff
@@ -18,11 +18,17 @@
 from .positioner import PositionerBase
 from .device import (Device, Component as Cpt)
 from .status import wait as status_wait
+from enum import Enum
 
 
 logger = logging.getLogger(__name__)
 
 
+class HomeEnum(str, Enum):
```

Comment:
To support who wants to use `HomeEnum.forward` and just  `"forward"` (the same for reverse).
I'm accepting suggestions here! =)